docs/tags.md

```diff
@@ -0,0 +1,37 @@
+---
+sidebarDepth: 2
+---
+
+## Tag Management
+
+The tag list can be viewed at the bottom left of each entry page.
+
+![Example entry, with one hidden under a spoiler toggle.](./img/tag_example.png)
+
+Tags are not part of the submission system, but still very important. If you do submit any new entries, it would be massively appreciated if you could add tags once the entry is approved, marking some as spoilers if needed.
+
+Tags for each entry are entirely user-managed, with all users being able to add tags and vote on their relevance to the work in question. The results can then be filtered through in the ‘Browse’ section of the website.
+
+If you wish to submit a new tag, all you have to do is type in the name and submit - it’ll then be looked into and approved or denied.
+
+::: tip
+Tags can be marked as a spoiler using the exclamation mark icon, or removed with the trash icon. This feature is not available to standard users.
+:::
+
+::: warning
+The power of each user's vote is limited to restrict abuse. If you feel that tags have been added incorrectly, contact a [moderator](./#moderator-list).
```

Comment:
```suggestion
The power of each user's vote is limited to restrict abuse. If you feel that tags have been added incorrectly, submit an edit to the entry where you simply comment in the [Submission Sources & Notes](./#submission-sources-notes) box why the tag should be removed. 
```

Reply:
Mods shouldn't be directly contacted for tag changes. The submission system supports people submitting just a comment with no changes, and that would be preferred over users contacting mods.